bundles/org.openhab.binding.renault/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -96,6 +98,12 @@
 		<label>Battery Energy Available</label>
 		<state pattern="%.1f %unit%" readOnly="true"></state>
```

Comment:
I reverted this back to unit as the 3.4.0-SNAPSHOT uses kWh as a default and does not convert to Joules.